### ISSUE TITLE
Restructuring: Cleanup and reconcile shared attribute definitions used in the docs

### ISSUE THREAD

**Issue by dedemorton** [elastic/beats#6144]
We currently have all the global Beats attributes defined in a shared file [here](https://github.com/elastic/beats/blob/master/libbeat/docs/shared-beats-attributes.asciidoc).

Some of these attributes are essentially duplicates of the shared attributes [here](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc). 

Since the shared attributes were created after the Beats documentation, there's a mismatch with the names.

Any attributes that are common to ALL docs (and not just Beats) should be moved to the shared file and the names should be reconciled (and replaced throughout the Beats docs).

I'm waiting for a resolution on the following issue before proceeding with these changes: https://github.com/elastic/docs/issues/220

**Comment by dedemorton** [comment]
Also look at `branch` and `doc-branch` in the version file. The x-pack topics included in beats require the `branch` attribute. If the usage of `branch` is identical to `doc-branch`, we should standardize on `branch`.

**Comment by dedemorton** [comment]
@bmorelli25 FYI in case you notice some weirdness in the way we've defined asciidoc attributes. I was hoping to have unresolved attribute handling added to the doc build before proceeding with this change because right now unresolved attributes fail silently. Might have to go ahead, though, because I've been waiting on this for a very long time.

**Comment by dedemorton** [comment]
Closed by #10956
